src/loaders/FileLoader.js

```diff
@@ -267,4 +278,4 @@ class FileLoader extends Loader {
 }
 
 
-export { FileLoader };
+export { FileLoader, HttpError };
```

Comment:
Since `HttpError` is only used inside `FileLoader`, it's probably better to not export the class right now.